p2p/simulations/network.go

```diff
@@ -392,6 +400,18 @@ func (self *Network) GetNodes() (nodes []*Node) {
 	return nodes
 }
 
+// GetNodes returns the existing nodes that are up
+func (self *Network) GetUpNodes() (nodes []*Node) {
```

Comment:
I don't see this used anywhere.

Reply:
yes sorry it was used by the dynamic discovery test. And in general this is a needed method

Reply:
OK fair enough.